modules/acid/src/main/ts/ephox/acid/api/colour/Transformations.ts

```diff
@@ -32,8 +32,15 @@ const anyToHex = (color: string): Hex =>
       return HexColour.fromRgba(RgbaColour.rgbaColour(r, g, b, a));
     });
 
+const anyToHexString = (color: string): string =>
```

Comment:
This isn't converting anything to hex, it's only converting RGBA/RGB to Hex, so can re please rename this to `rgbaToHexString` or similar? I'm also not sure this should be returning the original color as it might not be a hex value.

Reply:
This needs an acid changelog entry to say the new function has been added.

- [ ] Add acid changelog entry

Reply:
ah, the joys of this task changing so often during development. One of the places we changed was indeed 'any to hex', so we started by replicating that, then realised it wasn't viable anywhere except the one place that really needed it so had to roll back to just rgbToHex.